unst_msh_gen/plot_msh.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
```

Comment:
This is very similar to /scratch1/NCEPDEV/climate/Jessica.Meixner/UnstructuredExampleRun/plot_elementinfo_unstr.py 

we should work to add some of the comments and things in here.  If that was also your original inspiration, Steven Brus should also be credited as an author as I adopted my routines from him.

Reply:
So happy to hear you have sped things up!  Thank you for adding proper credit to Steven as well.  It's important to credit others when they have so kindly shared with us.  We should share back our changes with him! 

Do you also plan on adding the other comments and updates?  It'd be great to have the extra documentation if possible.

Reply:
Thank you for adding proper credit to Steven.  That is so important to acknowledge our collaborators and colleagues. 

Do you plan to add the extra comments from /scratch1/NCEPDEV/climate/Jessica.Meixner/UnstructuredExampleRun/plot_elementinfo_unstr.py ? I don't know if they were intntionally removed or perhaps an earlier version was your starting point?